keyboards/planck/keymaps/rootiest/keymap.c

```diff
@@ -0,0 +1,1920 @@
+        }
+        else SEQ_FOUR_KEYS(KC_P, KC_O, KC_O, KC_P) {
+            send_unicode_string("💩");
+            did_leader_succeed = true;
+        }
+        else SEQ_FOUR_KEYS(KC_B, KC_O, KC_A, KC_T) {
+            send_unicode_string("⛵");
+            did_leader_succeed = true;
+        }
+        leader_end();
+    }
+    // Run the wake-up RGB animation if performing wake-up
+    if (do_wake_animation) {
+        rgb_wakeup_sequence();
+    }
+}
+
+// Music mask controls
+
+bool music_mask_user(uint16_t keycode) {
+    switch (keycode) {
+        case RAISE:
+        case LOWER:
+            return false;
+        default:
+            return true;
+    }
+}
+
+void suspend_power_down_user(void) {
+    // Runs during start of system suspend
+    print("Going to sleep.");
+}
+
+void suspend_wakeup_init_user(void) {
+    // Runs during wake from system suspend
+}
+
+void keyboard_post_init_user(void) {
+    // Print welcome message to console
+    printf("Welcome to %s!\n", KEEB_MODEL_NAME);
+    // Read the user config from EEPROM
+    user_config.raw   = eeconfig_read_user();
+    do_wake_animation = user_config.do_wakeup_animation;
+    do_wake_audio     = user_config.do_wakeup_audio;
+    rgbcon_tracker    = user_config.rgbcon_tracker;
+
+    // Tell the console the status of saved config
+    if (user_config.do_wakeup_animation) {
+        print("Wake animation enabled.\n");
+    } else {
+        print("Wake animation disabled.\n");
+    }
+    if (user_config.do_wakeup_audio) {
+        print("Wake music enabled.\n");
+    } else {
+        print("Wake music disabled.\n");
+    }
+    switch (user_config.rgbcon_tracker) {
+        case 2:
+            print("RGB mode: Static White\n");
+            break;
+        case 3:
+            print("RGB mode: Static Cyan\n");
+            break;
+        case 4:
+            print("RGB mode: Breathing Lights\n");
+            break;
+        case 5:
+            print("RGB mode: Rainbow Swirl\n");
+            break;
+        case 6:
+            print("RGB mode: Rainbow Mood\n");
+            break;
+        default:
+            print("RGB mode: Disabled RGB\n");
+            break;
+    }
+
+    // Enable the LED layers
+    rgblight_enable_noeeprom();  // Enables RGB, without saving settings
+    rgblight_layers = my_rgb_layers;
+
+    /*
+     * Initialize the LED crawl wake animation here
+     * To perform it on just the first wake
+     */
+    wake_rgb_timer = timer_read();
+    waking_up      = true;
+
+    // Initialize OLED display
+    /*
+        if (USING_OLED_DISPLAY) {
+            print("Initializing display!\n");
+            render_logo();
+        }
+    */
+}
+
+// EEPROM is getting reset!
+void eeconfig_init_user(void) {
+    user_config.raw                 = 0;
+    user_config.do_wakeup_animation = false;  // We want this disabled by default
+    user_config.do_wakeup_audio     = false;  // We want this disabled by default
+    user_config.rgbcon_tracker      = 0;
```

Comment:
```suggestion
```

You don't need these if you're setting the raw to 0.  That will zero out all of the settings for the structure, so no need to explicitly re-zero them.